sdk-ext-lib/src/main/java/cash/z/ecc/sdk/fixture/Zip321UriBuildFixture.kt

```diff
@@ -0,0 +1,24 @@
+package cash.z.ecc.sdk.fixture
+
+import cash.z.ecc.sdk.model.WalletAddress
+import cash.z.ecc.sdk.model.Zatoshi
+import cash.z.ecc.sdk.model.ZecRequest
+import cash.z.ecc.sdk.model.ZecRequestMessage
+import kotlinx.coroutines.runBlocking
+
+object Zip321UriBuildFixture {
+    const val URI: String = "zcash:ztestsapling10yy2ex5dcqkclhc7z7yrnjq2z6feyjad56ptwlfgmy77dmaqqrl9" +
```

Comment:
@pacu I'm assuming `ztestsapling` at the beginning means testnet.  Does the Rust code have special consideration for the testnet versus mainnet when it parses addresses?  For example, if we initialize the SDK with Mainnet but pass it a Testnet address for send/receive, what happens?

I think this raises a question about whether our fixture values should be always mainnet, always testnet, or we have both.

I think I have a preference for either always mainnet or both.  The reason is that these fixture values will show up in our automatically generated screenshots, so having mainnet versions would mean that our screenshots more closely match what users would see in production.  (Although we use the screenshots for debugging right now, they'll also be useful for marketing or store listings because we can programmatically generate them for multiple languages easily so that's the motivation to have "production" data displayed in them).

Reply:
I think on Android you can load the FFI and assign a fixed environment to it and then pass it on the calls made to the rust layer. 

That same consistency counts for this case since URIs have parameters that are tied to the network (mainnet or testnet)